### ISSUE TITLE
One of the tests in queries/line_scan.py fails in parallel and is in the skip list.

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1028]
The tests ls_1_16 and ls_1_18 are both in the skip list from queries/line_scan.py. This should either be fixed or a mode specific baseline added. I am not sure which is best here.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 06/28/2010 07:40 pm
Original update: 07/22/2010 08:01 pm
Ticket number: 179
